python/apps/utils.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
A file similar to this should be packaged up with our python at some point.  Before we made the push for the inmem indices last week it was on my radar to do!  I want writers and readers and quality of life utility methods right in the library.